cli/src/commands/commit.rs

```diff
@@ -63,6 +63,9 @@ pub(crate) struct CommitArgs {
     /// The change description to use (don't open editor)
     #[arg(long = "message", short, value_name = "MESSAGE")]
     message_paragraphs: Vec<String>,
+    /// Open an editor to edit the commit's description
```

Comment:
Same here: I think it would be good to explain how this isn't just the default behavior.